dashboard/app/views/home/_survey_nps.html.haml

```diff
@@ -10,7 +10,7 @@
     cursor: auto;
   }
 
-- if show_survey?
+- if show_nps_survey?
```

Comment:
Do we need this check both here and in the home/index view? Can't it just be here?